package.json

```diff
@@ -87,5 +90,34 @@
   "homepage": "https://github.com/sequelize/cli",
   "engines": {
     "node": ">=10.0.0"
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-angular"
```

Comment:
Any specific reason why you use this configuration?

Reply:
it's a copy from sequelize/sequelize